ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.model.js

```diff
@@ -56,11 +62,20 @@ module.exports = Backbone.AssociatedModel.extend({
     const lat = this.get('mouseLat')
     const lon = this.get('mouseLon')
     const dms = `${mtgeo.toLat(lat)} ${mtgeo.toLon(lon)}`
+    const mgrs =
```

Comment:
This kind of logic feels like it could be located somewhere more general

Reply:
Maybe in the next iteration and make the whole thing a React component (@Lambeaux 's idea).

Reply:
TODO comment has been added since, best I can tell, we'd need to release a version with API or behavior updates. 
https://codice.atlassian.net/browse/DDF-4335